src/current/_includes/releases/v23.1/v23.1.3.md

```diff
@@ -0,0 +1,143 @@
+- Fixed a problem that could lead to erroneously refused lease transfers, causing the following error message: `refusing to transfer lease to [...] because target may need a Raft snapshot: replica in StateProbe`. [#103849][#103849]
+- Fixed a bug where [`COPY`](../v23.1/copy-from.html) in v23.1.0 and beta versions would incorrectly encode data with multiple [column families](../v23.1/column-families.html). The data must be dropped and re-imported to be encoded correctly. [#103393][#103393]
+- Fixed a bug where running a [`debug` command](../v23.1/cockroach-commands.html#commands) that manipulates a store (e.g., `debug compact`) without first terminating the node using the store, could result in corruption of the node's store if encryption-at-rest was enabled. [#102877][#102877]
+- Fixed a bug where [`SHOW DEFAULT PRIVILEGES`](../v23.1/show-default-privileges.html) did not work correctly if the database name or schema name being inspected had upper-case or special characters. [#103952][#103952]
+- Fixed a bug that could cause queries with [joins](../v23.1/joins.html) or [subqueries](../v23.1/subqueries.html) to omit rows where column values are `NULL` in very rare cases. This bug was present since v20.2. [#103803][#103803]
+- Fixed a bug that could cause goroutines to hang during [SCRAM authentication](../v23.1/security-reference/scram-authentication.html). [#104197][#104197]
+- The `sys.cpu.combined.percent-normalized` metric now uses `GOMAXPROCS`, if lower than the number of CPU shares when calculating CPU utilization. [#104191][#104191]
+- The warning message about a missing `--advertise-addr` flag is no longer displayed when the flag is specified on server start. [#104227][#104227]
+- Fixed a rare bug where stale multi-column table [statistics](../v23.1/cost-based-optimizer.html) could cause table statistics forecasts to be inaccurate, leading to un-optimal query plans. [#104229][#104229]
+- Fixed a bug in v23.1.0 where the `node_id` field would be omitted from logs. The `node_id` value has now been restored to the logs. [#103798][#103798]
+- Fixed a bug where the collection of `KV bytes read` and `KV gRPC calls` execution statistics during [`EXPLAIN ANALYZE`](../v23.1/explain-analyze.html) could be incorrect (it would remain at zero) in some cases. The bug was introduced in the v23.1.0 release. [#104194][#104194]
+- Fixed an incorrect results bug where predicates on [computed columns](../v23.1/computed-columns.html) are derived when an ORed predicate on a column in the computed column expression is present. This bug only affects CockroachDB when the `optimizer_use_improved_computed_column_filters_derivation` [session setting](../v23.1/set-vars.html) is `true`. It currently defaults to `false` in releases v23.1.2 and v22.1.10. This bug does not affect releases prior to these versions. [#104276][#104276]
```

Comment:
```suggestion
- Fixed an incorrect results bug in v22.1.11 and 23.1.3 and above when the `optimizer_use_improved_computed_column_filters_derivation` [session setting](../v23.1/set-vars.html) is `true`. Predicates on [computed columns](../v23.1/computed-columns.html) are derived when an ORed predicate on a column in the computed column expression is present. [#104276][#104276]
```
WDYT about this edit? I'm still not sure that we should mention 22.1 in this note (if it will be fixed in a 22.1.x version) and I'm not 💯 this is still accurate to the intent. I still don't love ORed but also not sure if my suggestion for that bit is accurate either.